### PR TITLE
Adds a 404 template

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:
Adds a 404 template for Archeo
Screenshot 2022-02-09 at 14 15 19

#### Related issue(s):
https://github.com/Automattic/themes/issues/5426

### CODE REVIEW

**Review by mikachan — COMMENTED** [review]
This looks good, I like the big heading.

The search box looks pretty small at mobile:

image

I think this is the 50% width setting. It looks OK to me at around 70% instead, but then this means it's also bigger at larger resolutions too.

File: archeo/inc/patterns/hidden-404.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * 404 content.
+ */
+return array(
+	'title'    => __( '404 content', 'archeo' ),
+	'inserter' => false,
+	'content'  => '<!-- wp:heading {"style":{"typography":{"fontSize":"clamp(4rem, 40vw, 20rem)","fontWeight":"200","lineHeight":"1"}},"className":"has-text-align-center"} -->
```
Comment (kjellr) [inline archeo/inc/patterns/hidden-404.php]: Let's change this weight to `100`.

**Comment by kjellr** [comment]
This looks good! Just two small changes: 

- Let's switch to the 100% wide search field.
- Please add a `1em` spacer block above the search field, and a `2em` spacer block below the search field. This will help even out the space it all takes up. 

That'll look like this in the end: 

Screen Shot 2022-02-10 at 10 34 09 AM

Thank you!

**Comment by mikachan** [comment]
Thanks! I've made the above changes.

**Review by jffng — APPROVED** [review]
LGTM!

**Pull request merged**